app.js

```diff
@@ -224,6 +225,11 @@ app.put('/unpublish',
         middleware.validateRequest(["description", "dateUpdated", "public"]),
         routes.unpublish);
 
+app.get('/remix/:projectId',
+        middleware.setUserIfTokenExists,
+        middleware.publishLogin(true),
```

Comment:
This looks wrong to me. How can someone remix if they aren't logged into thimble?